### PR TITLE
Add documentation on Arcus.Web.Api.Logging.ExceptionHandlingMiddleware

### PR DESCRIPTION
**Pull request by fgheysels**
Extended the readme.md with some documentation on the `ExceptionHandlingMiddleware` class (what it is and how to use it).

#14 #16

### CODE REVIEW

**Comment by fgheysels** [comment]
I don't know why all those commits are listed in this PR, since this PR actually contains just 3 or 4 commits and has 2 changed files.
(I've merged the changes from the arcus base repo into my fork, then created a new branch, did my changes, merged that branch into the master branch of my fork, and now PR my fork into the original arcus repo)

**Comment by tomkerkhove** [comment]
> I don't know why all those commits are listed in this PR, since this PR actually contains just 3 or 4 commits and has 2 changed files.
> (I've merged the changes from the arcus base repo into my fork, then created a new branch, did my changes, merged that branch into the master branch of my fork, and now PR my fork into the original arcus repo)

No problem!

**Comment by fgheysels** [comment]
> The docs look good but would you mind moving it to the published docs please?
> 
> For example we have a `index.md` which serves as the landing page and document features in a sub-page, for example [Arcus.Security](https://github.com/arcus-azure/arcus.security/blob/master/docs/index.md).
> 
> My suggestion would be to `/features/logging`.

Did some changes to (hopefully) adhere to the convention.

File: docs/features/log-unhandled-exceptions.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Arcus.WebApi.Logging.ExceptionHandlingMiddleware
```
Comment (tomkerkhove) [inline docs/features/log-unhandled-exceptions.md]: Can you add the Jekyll header please?

Think we can call this page `Logging`

Comment (fgheysels) [reply docs/features/log-unhandled-exceptions.md]: Like this ?

**Comment by tomkerkhove** [comment]
Thanks @fgheysels!

**Pull request merged**